sandbox.html

```diff
@@ -70,11 +70,11 @@
 
     const cadenzaClient = cadenza(location.origin + '/trunk', { iframe: 'iframe', debug: true });
     const actionHandlers = {
-      show ({ embeddingTargetId, hideMainHeaderAndFooter, hideWorkbookToolBar, mediaType }) {
+      show ({ embeddingTargetId, hideMainHeaderAndFooter, hideWorkbookToolBar, jasperReportAsPdf }) {
         cadenzaClient.show(embeddingTargetId, {
           hideMainHeaderAndFooter: (hideMainHeaderAndFooter === 'on'),
           hideWorkbookToolBar: (hideWorkbookToolBar === 'on'),
-          accept: mediaType
+          ...(jasperReportAsPdf && { mediaType: 'application/pdf' })
```

Comment:
My comment was only about the sandbox. Here, I'd stick to the more future-proof mediaType, because this param is published is directly related to the endpoint param.

(Also the example diverges from this API atm.)

Reply:
What do you mean with "stick to the more future-proof mediaType". I changed the option name from "accept" to "mediaType" as suggested in this comment: https://github.com/DisyInformationssysteme/cadenza.js/pull/5#discussion_r1338301472

And I don't find the error in the example in README.md. The only difference to here that I see is, that in the example I left out the optional options "hideMainHeaderAndFooter" and "hideWorkbookToolBar".

Reply:
Sry, my bad, we _are_ in sandbox code here 🤦

Still, you need to check for ==='on', otherwise the mediaType will always be set.

Reply:
Turns out that jasperReportAsPdf is undefined when the checkbox is not checked. So it worked correctly. But it is still nicer to check for "on" explicitly, so I changed it.